e2e-tests/tests/delegator_rewards/test_spo_journey.py

```diff
@@ -202,3 +203,26 @@ def test_spo_participation(
 @mark.xdist_group("block_participation")
 def test_no_unexpected_producers(block_participation):
     assert not block_participation["producer_participation"], "Unexpected producer participation data"
+
+
+class TestMarginFee:
+    @fixture(scope="class")
+    def random_margin_fee(self) -> int:
+        return random.randint(1, 10000)
```

Comment:
IDK about inclusivity but 0 and 10000 are valid values

Reply:
both included, fixing